chainerx_cc/chainerx/python/routines.cc

```diff
@@ -596,6 +596,14 @@ void InitChainerxMath(pybind11::module& m) {
           py::arg("x"),
           py::arg("axis") = nullptr);
     m.def("sqrt", [](const ArrayBodyPtr& x) { return MoveArrayBody(Sqrt(Array{x})); }, py::arg("x"));
+    m.def("pow",
+      [](const ArrayBodyPtr& x1, const ArrayBodyPtr& x2) { return MoveArrayBody(Pow(Array{x1}, Array{x2})); },
+      py::arg("x1"),
+      py::arg("x2"));
+    m.def("pow",
```

Comment:
You are right about `numpy.power`. Sorry but could you change it back to `chainerx.power` and similarly `Power` to the C++ API?